### ISSUE TITLE
Explore incorporating Jest Axe

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#207]
This also looks promising - [Jest Axe](https://github.com/nickcolley/jest-axe) - this will allow people to run Axe against components in Jest. Not whole pages, but they can at least catch some things at the testing layer.

I would be interested to see if color contrast violations are captured.
